network/config/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "config"
+version = "0.1.0"
+authors = ["Allen Clement <aclement@gmail.com>"]
```

Comment:
well I suspect that this is just for testing, maybe?  these should just be copypasta of others due to CLA, afaiu.

Reply:
+1.